<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table">
     <column>
      <property name="text">
       <string>login</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>isadmin</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_button">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="edit_button">
     <property name="text">
      <string>Edit User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string>Delete User</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="back_button">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
